app/views/follow_requests/destroy.js.erb

```diff
@@ -0,0 +1,8 @@
+console.log("TEST")
+
+
+var unfollow_link = $(".<%= dom_id(@follow_request) %>")
+
+var followForm = $("<%= j(render "follow_requests/form", follow_request: @follow_request.recipient.received_follow_requests.build) %>")
```

Comment:
That's weird that this piece of code
`follow_request: @follow_request.recipient.received_follow_requests.build` doesn’t work for my app.
I replaced it with `sender: @follow_request.sender, recipient: @follow_request.recipient)`.
May be Aldo could explain :)

Reply:
@Ritumka it could be because defined associations or different model names.